src/assets/_project/_blocks/components/accordion/accordion.js

```diff
@@ -50,6 +50,15 @@
     hashTrigger();
     window.onhashchange = hashTrigger;
 
+    // inserting tab index dynamically
```

Comment:
This change is not merged correclty . If you see branch 3.1.10 

https://github.com/qld-gov-au/qg-web-template/blob/release-v3.1.10/src/assets/_project/_blocks/components/accordion/accordion.js